<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Join</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>    AHO</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Credit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Splite</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="7">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete Node</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Key to Delete</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>100</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>250</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Go To View 2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>220</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Go To View 1</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>220</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="radioButton_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 2</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 3</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="radioButton_12">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_13">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 2</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_14">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 3</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Splite</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>splite the Tree 1 to :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>100</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Splite</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>250</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Go To View 2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>220</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Go To View 1</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>220</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="radioButton_15">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_16">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 2</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_17">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 3</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Tree 1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Tree 2</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Insert Node</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>101</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>101</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>120</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>120</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>cArray</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>iValue</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>90</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>fValue</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>90</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Key</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Random number</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Ivalue</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_12">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Push</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_17">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>250</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Go To View 2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_13">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>220</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Go To View 1</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 2</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 3</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>220</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="radioButton_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 2</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 3</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Search of Node (Key)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>100</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>100</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="radioButton_18">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>View 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_19">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>View 2</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="radioButton_21">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_22">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 2</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_23">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tree 3</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6">
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>281</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Join the (Tree 1) , (Tree 2) on (main Tree)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Join</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_20">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>100</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Join</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_18">
       <property name="geometry">
        <rect>
         <x>298</x>
         <y>220</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Go To Tree 1 View 1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_19">
       <property name="geometry">
        <rect>
         <x>298</x>
         <y>250</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Go To Tree 2 View 2</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_7">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
